core/client/verify_test.go

```diff
@@ -82,85 +58,16 @@ func sign(s crypto.Signer, smr *trillian.SignedMapRoot) *trillian.SignedMapRoot
-				Committed: nil,
-				LeafProof: &trillian.MapLeafInclusion{
-					Leaf:      &trillian.MapLeaf{},
-					Inclusion: make([][]byte, 256),
-				},
-				Smr: sign(signer, &trillian.SignedMapRoot{
-					TimestampNanos: 1506524755543208185,
-					RootHash:       []byte{0x0e, 0xfc, 0x54, 0xad, 0xe0, 0xfc, 0xe8, 0x76, 0x55, 0x8c, 0x97, 0x38, 0xf5, 0xaa, 0x89, 0xe4, 0xd9, 0x9c, 0x0b, 0x8b, 0x6f, 0xe0, 0xb6, 0x2d, 0xbf, 0x63, 0x59, 0xcf, 0xc2, 0xad, 0xbb, 0xd7},
-					MapId:          9175411803742040796,
-					MapRevision:    1,
-					Metadata:       mustMetadataAsAny(t, &pb.MapperMetadata{}),
-				}),
-			},
-			wantErr: false,
-		},
-		{
-			desc:    "Tree size 2",
-			userID:  "nocarol",
-			appID:   "app",
-			trusted: trillian.SignedLogRoot{},
-			in: &pb.GetEntryResponse{
-				VrfProof:  []byte{0x9f, 0x8f, 0xb1, 0x41, 0xbc, 0x10, 0xcb, 0xe2, 0x02, 0xe8, 0x3e, 0x8a, 0xe2, 0xd0, 0xe7, 0xe3, 0xc9, 0xa2, 0x83, 0x94, 0x85, 0xf7, 0xca, 0x8f, 0x33, 0xb6, 0x52, 0x56, 0xb0, 0x76, 0x8e, 0xf9, 0x6e, 0x0c, 0x8a, 0x1c, 0xe6, 0x7d, 0x8b, 0xb6, 0x73, 0xb3, 0xae, 0x51, 0x36, 0x52, 0xab, 0x2b, 0x9d, 0x5a, 0x96, 0xdd, 0xae, 0x2a, 0x74, 0x74, 0x02, 0x6b, 0xdd, 0x16, 0x86, 0x70, 0x94, 0x15, 0x04, 0x7b, 0xcd, 0x07, 0x03, 0xb9, 0x69, 0xe3, 0x72, 0x35, 0xdb, 0xfc, 0xb2, 0xa3, 0x4c, 0x22, 0x6b, 0xaa, 0xce, 0x92, 0x6b, 0xcf, 0x02, 0x11, 0x78, 0x7b, 0x1f, 0x5c, 0x2f, 0xff, 0xb9, 0x34, 0x32, 0xa1, 0xd9, 0xba, 0xec, 0xa5, 0x9d, 0x5e, 0xa6, 0xbb, 0xb6, 0x77, 0x92, 0x4c, 0x0c, 0x2d, 0x76, 0xdf, 0xbe, 0x9e, 0xa0, 0x93, 0xde, 0xf5, 0xa1, 0xc1, 0x4e, 0x9e, 0x19, 0x39, 0x16, 0xfe, 0x60},
-				Committed: nil,
-				LeafProof: &trillian.MapLeafInclusion{
-					Leaf: &trillian.MapLeaf{},
-					Inclusion: append(make([][]byte, 255), []byte{
-						92, 215, 13, 113, 97, 138, 214, 158, 13, 29, 227, 67, 236, 34, 215, 4, 76, 188, 79, 247, 149, 223, 227, 147, 86, 214, 90, 126, 192, 212, 113, 64,
-					}),
-				},
-				Smr: sign(signer, &trillian.SignedMapRoot{
-					TimestampNanos: 1506596629587264426,
-					RootHash:       []byte{0x2c, 0x27, 0x03, 0xe0, 0x34, 0xf4, 0x00, 0x2f, 0x94, 0x1d, 0xfc, 0xea, 0x7a, 0x4e, 0x16, 0x03, 0xee, 0x8b, 0x4e, 0xe3, 0x75, 0xbd, 0xf8, 0x72, 0x5e, 0xb8, 0xaf, 0x04, 0xbf, 0xa3, 0xd1, 0x56},
-					MapId:          2595744899657020594,
-					MapRevision:    2,
-					Metadata:       mustMetadataAsAny(t, &pb.MapperMetadata{HighestFullyCompletedSeq: 1}),
-				}),
-			},
-		},
-	} {
-		err := v.VerifyGetEntryResponse(ctx, domainID, tc.appID, tc.userID, tc.trusted, tc.in)
-		if got, want := err != nil, tc.wantErr; got != want {
-			t.Errorf("VerifyGetEntryResponse(%v, %v, %v, %v): %t, wantErr %t (err=%v)",
-				tc.userID, tc.appID, tc.trusted, tc.in, got, want, err)
-		}
+	for _, tc := range getEntryResponses {
```

Comment:
where is getEntryResponses declared?  It seems not to be in this set of commits.
Did you just move the previous version's test table to a different file?

Reply:
I've moved `getEntryResponses` to `vectors_test.go` to keep the *large* test vectors separate from code.

Reply:
Sorry I failed to notice |vectors_test.go| somehow.

Incidentally I was under the impression that one should only name files |foo_test.go| if they contain actual `func TestFoo(t *testing.T)` material
and that in cases like this (test support files) one would name them eg. |test_vectors.go| or make a directory `testonly` and then add `testonly/vectors.go`

One for a future PR if you want to change it.

Reply:
Good point. I think you're right about this.  However, it also feels a bit strange to put a lot of distance between the test that is using the test vectors and the vectors themselves.